doc/integrator/c2cwsgiutils.rst

```diff
@@ -12,4 +12,6 @@ c2cwsgiutils and c2cgeoportal services.
 
 To use the protected tools, you should at least set ``C2C_SECRET`` to a secret password in the ``env.project`` file.
 
+Note that there is no real protection against brute-force attack on those tools.
```

Comment:
I really don't think we should state that so explicitly in a documentation available on the Internet

Reply:
Maybe we can move it to env.project, before C2C_SECRET line, but it will still be in a public code repository.
@sbrunner also talked about https://camptocamp.github.io/c2cgeoportal/master/integrator/security.html

Reply:
Ok removed.